azure-pipelines.yml

```diff
@@ -116,3 +116,63 @@ jobs:
       fi
     displayName: 'Running benchmarks'
     condition: true
+
+- job: 'Docs'
+  pool:
+    vmImage: ubuntu-16.04
+  timeoutInMinutes: 90
+  steps:
+  - script: |
+      echo '##vso[task.setvariable variable=CONDA_ENV]pandas-dev'
+      echo '##vso[task.setvariable variable=ENV_FILE]environment.yml'
```

Comment:
What environment file is this?

Reply:
The one in the root, for dev environments (like locally). It's the one that contains the dependencies to build the docs.

Reply:
Oh of course. Should we re-use `ci/deps/travis-36-doc.yaml` for now (and rename that when the travis doc job is removed)?

Reply:
I thought travis was using `environment.yml`.

I'd personally use `environment.yml`, so we build in the CI with the same versions as locally, and remove `travis-36-doc.yaml` when we remove the travis job. Also it keep things simpler.

Any other advantage of using a specific dependencies file besides installing only the needed packages for the docs?

Reply:
> besides installing only the needed packages for the docs?

I think that's the primary benefit, but it's not a big deal to install extra deps (like test-only ones).

Stepping back, my main request is that we use the same env for building the travis & azure docs. I don't have a strong preference for whether that's `/environment.yaml` or `ci/deps/travis-36-doc.yaml`.

Reply:
+1 for same on each

+1 for environment.yml

Reply:
That makes sense, I changed travis build to use `environment.yml` for the docs.